linode_api4/objects/serializable.py

```diff
@@ -71,6 +72,13 @@ class JSONObject(metaclass=JSONFilterableMetaclass):
     are None.
     """
 
+    put_class: ClassVar[Optional[Type["JSONObject"]]] = None
+    """
+    An alternative JSONObject class to use as the schema for PUT requests.
+    This prevents read-only fields from being included in PUT request bodies,
+    which in theory will result in validation errors from the API.
+    """
```

Comment:
I added this specifically to prevent the read-only `ConfigInterface.ipv6.slaac.address` field from being sent along in PUT requests, which would in theory result in a validation error.